### DISCUSSION TITLE
Your Copilot experience is not fully configured, complete your setup

### DISCUSSION THREAD

**Discussion by dmwyatt** [community/community#19068] [Copilot Conversations]
I'm getting the following in PyCharm:

![image](https://user-images.githubusercontent.com/163359/174864848-44956ad3-356c-49cb-982c-3c0b620b84ba.png)

Clicking "Copilot Settings" doesn't do anything.  Additionally, CoPilot just isn't working at all.

**Comment by rauzerfabiano** [comment]
Maybe this solution can help you:
https://github.com/orgs/github-community/discussions/19061#discussioncomment-2995893

The solution in the discussion says the following:
1. Go to https://github.com/github-copilot/tp_signup
2. Press the button [Continue without payment details.](https://github.com/github-copilot/signup/success)
3. In Suggestions matching public code; select Allow or Block.
4. Press Save and get started
5. Close or Reload the IDE or Code Editor

**Comment by 201flaviosilva** [comment]
GitHub Copilot is now paid 🤑 : https://github.blog/2022-06-21-github-copilot-is-generally-available-to-all-developers/

**Comment by lso2** [comment]
I had to check some boxes on my page on github account page, which wasn't intuitive and which you have to scroll down the the bottom to find. After selecting the options, then it allowed me to proceed.
